fancy_cache/memory.py

```diff
@@ -12,6 +15,10 @@
 
 LOGGER = logging.getLogger(__name__)
 
+COMPRESS_REMEMBERED_URLS = getattr(
+    settings, "FANCY_COMPRESS_REMEMBERED_URLS", False
+)
```

Comment:
Nice! So it's entirely optional by default. That makes it feel safer to land this.